### ISSUE TITLE
Automation NOT adding issues in wp-proudcity to new project board

### ISSUE THREAD

**Issue by curtismchale** [proudcity/wp-proudcity#2568]
## Source

The automation DOES add new issues in `wp-proudcity` to our private board but DOES NOT add issues to our [new public project board](https://github.com/orgs/proudcity/projects/4).

@lukefretwell can you add an automation to get these added to the backlog automatically?

**Comment by lukefretwell** [comment]
@curtismchale can you test?

What I've done for both:

* Labeled/color-code columns the same
* Removed no status column on public/private board, moved them to new backlog column

Should we sync the workflows so that they're the same?

- https://github.com/orgs/proudcity/projects/3/workflows/16922575
- https://github.com/orgs/proudcity/projects/4/workflows/21417057

@kevindherman the backlog order got messed up so feel free to re-order prioritized ones.

**Comment by lukefretwell** [comment]
One item I'm not sure of: Are we managing both boards so that items are in the correct columns?

**Comment by curtismchale** [comment]
@lukefretwell yes I do try to make sure I move things for each board so it stays updated for customers.

**Comment by lukefretwell** [comment]
@curtismchale this is set up just like for private board, not sure why it wouldn't auto-add. Am I missing something?

https://github.com/orgs/proudcity/projects/4/workflows/21417057
